remake_depth.yml

```diff
@@ -40,19 +40,21 @@ targets:
     command: depth_link_mnlakelist("data/depth_data_raw/depth_mn_lakelist.tsv")
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
+      #- data/NHD_state_crosswalk/nhd2dowlknum.RData
     
   data/depth_data_linked/depth_herb_jacobson.tsv:
     command: depth_link_jacobson("data/depth_data_raw/depth_Herb_Jacobson.csv")
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
+      #- data/NHD_state_crosswalk/nhd2dowlknum.RData
     
   data/depth_data_linked/depth_lagos.tsv:
     command: depth_link_lagos(necsc_config)
     depends:
       - data/NHD_shape_large/NHDWaterbody.dbf
     
   data/depth_data_linked/depth_swimms_wisconsin.csv:
-    command: depth_link_swimms("data/depth_data_raw/wisconsin_swims_lakes_depth.tsv")
+    command: depth_link_swimms("data/depth_data_raw/wisconsin_swims_lakes_depth.tsv", "data/NHD_state_crosswalk/nhd2wbic.RData")
```

Comment:
I mean, should it be in the remake now that its part of the overarching build process?